lib/trace-span.js

```diff
@@ -19,17 +19,28 @@
 // TODO(ofrobots): replace this file with the protobuf
 
 /**
+ * Converts the provided time in millis into a seconds/nanos
+ * style timestamp.
+ */
+var getTimestamp = function(time) {
+  return {
+    seconds: Math.floor(time / 1000),
+    nanos: Math.floor((time % 1000) * 1000000)
+  };
+};
+
+/**
  * Creates a trace span object.
  * @constructor
  */
 function TraceSpan(name, spanId, parentSpanId, time) {
```

Comment:
I agree. I can get rid of our structural classes now if we don't mind making this CL larger. Or I can do it in the next one.